README.md

```diff
@@ -101,9 +101,11 @@ We currently test Linux (Ubuntu and scratch), MacOS and Windows as packaged by
 * Interpreter
   * Linux is tested on amd64 (native) as well arm64 and riscv64 via emulation.
   * MacOS and Windows are only tested on amd64.
+  * NetBSD and OpenBSD are only tested on amd64.
 * Compiler
   * Linux is tested on amd64 (native) as well arm64 via emulation.
   * MacOS and Windows are only tested on amd64.
+  * FreeBSD is only tested on amd64.
```

Comment:
wondering if Net/OpenBSD passes with compiler or not https://github.com/tetratelabs/wazero/blob/dc08732e57d575a240774bc64e87cbed59f81c34/config_supported.go#L8 ?

Reply:
Yes. NetBSD works. I was keeping that feature change for after I have working tests.
OpenBSD fails for some unknown reason.

One of the reasons I am holding the change, is that illumos _also_ works, and I wanted to add CI for it _before_ enabling the compiler for both.

wazero is (as a dependency) on [pkgsrc](https://pkgsrc.org/) which is the package manager for NetBSD and illumos, so it'd be nice to enable the compiler there.

Reply:
ok, let's keep it as-is for now, and later enable compilers there when you have time